src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -158,44 +105,6 @@ define(function (require, exports, module) {
         },
 
         /**
-         * Scrolls the layers panel to make (newly) selected layers visible.
-         *
-         * @param {LayerStructure} layerStructure
-         */
-        _scrollToSelection: function (layerStructure) {
```

Comment:
The behavior remains unchanged. The first newly selected LayerFace will scroll into view unless it is directly selected by user from the LayerPanel